lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -971,6 +971,24 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
             rect.bottom,
           );
       }
+
+      if (rect.top < 0) {
+        final shiftAmount = rect.top;
+        rect = Rect.fromLTRB(rect.left,
+          0,
+          rect.right,
+          rect.bottom - shiftAmount,
+        );
+      }
+
+      if (rect.bottom > viewSize.height) {
```

Comment:
I think it is impossible to have a tooltip that passes the bottom border
because we show tooltips on top of points.
we can remove this section.

Reply:
When you have multiple series going below the chart area it's possible

Reply:
Actually, even something like this is enough to prove the point
![image](https://user-images.githubusercontent.com/4688431/76624312-54498d80-6535-11ea-90eb-4445de9a5ebe.png)

Reply:
Okay fair enough :)